### PR TITLE
Merge dev-production to main

### PR DESCRIPTION
**Pull request by stratofax**
The last round of changes, approved by the client, posted to the public site: https://erlenepsyd.com/

### CODE REVIEW

File: erlenepsyd.com/pelicanconf.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
+from typing_extensions import TypeVarTuple
+
 AUTHOR = "Erlene Rosowsky"
 SITENAME = "the older self"
 # pelican-seo requires a site URL
 SITEURL = "https://erlenepsyd.com"
 PATH = "content"
 # Set to False before deploying to production
-RELATIVE_URLS = False
+RELATIVE_URLS = True
```
Comment (flexicon33) [inline erlenepsyd.com/pelicanconf.py:9]: I see, do you?

**Review by flexicon33 — APPROVED** [review]
Looks good

**Comment by stratofax** [comment]
Merged to main

File: erlenepsyd.com/pelicanconf.py (hunk 1 above)
Comment (stratofax) [reply erlenepsyd.com/pelicanconf.py:9]: Yes, resolved.

**Pull request merged**